src/main/java/seedu/address/model/person/Person.java

```diff
@@ -98,6 +98,11 @@ public boolean isSamePerson(Person otherPerson) {
             && otherPerson.getName().equals(getName());
     }
 
+    @Override
+    public int compareTo(Person o) {
+        return this.getName().fullName.compareTo(o.getName().fullName);
```

Comment:
I'm not sure if this violates the [Law of Demeter](https://nus-cs2103-ay2223s1.github.io/website/se-book-adapted/chapters/principles.html#law-of-demeter)